ui/components/ui/definition-list/definition-list.stories.js

```diff
@@ -34,39 +28,83 @@ const tooltips = {
   Ticker: 'The currency symbol of the primary currency for this network',
 };
 
-export const DefaultStory = () => (
-  <DefinitionList
-    dictionary={object('dictionary', basic)}
-    gapSize={select('gapSize', SIZES, SIZES.SM)}
-  />
+export default {
+  title: 'Components/UI/DefinitionList',
+  id: __filename,
+  argTypes: {
+    dictionary: { control: 'object', name: 'Dictionary' },
+    gapSize: {
+      control: 'select',
+      name: 'Gap Size',
+      options: SIZES,
+      defaultValue: SIZES.SM,
```

Comment:
options arg type control should to be an array
```suggestion
      options: Object.values(SIZES),
      defaultValue: Object.values(SIZES.SM),
```